lib/model/card/action/scry_action.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+import 'card_action.dart';
+
+class ScryAction extends CardAction {
+  List<Resource> cost;
+  int numToScry;
+
+  ScryAction(description, List<Resource> cost, int numToScry) : super(cost, description);
```

Comment:
can do `ScryAction(String description, List<Resource> cost, this.numToScry) : super(cost, description)` - which will result in the `numToScry` actually being set

Reply:
Makes sense, fixed.